yaml/pois.yaml

```diff
@@ -265,9 +265,10 @@ filters:
   - filter: {amenity: toilets}
     min_zoom: 18
     output: {kind: toilets}
-  - filter: {barrier: gate}
+  - filter: {barrier: [gate, toll_booth]}
```

Comment:
While this works, I propose two separate filters for `gate` and `toll_booth` as the gate min_zoom is likely far too early (!?) and should be modified to something like zoom 17, setting both the kind values explicitly for each filter.